### DISCUSSION TITLE
希望可以增加webdav功能

### DISCUSSION THREAD

**Discussion by soft98-top** [gaowei-space/markdown-blog#21] [Ideas]
博客展示小而美我觉得大佬的项目很棒，但是我觉得应该提供一个上传的通道，比如webdav，本地可以直接远程连接，这样就可以方便更新和发布文章，可参考go语言写的webdav实现，https://github.com/hacdias/webdav。

**Comment by gaowei-space** [comment]
👌🏻 研究下~

**Comment by soft98-top** [comment]
这两天试了一下在服务器上启用webdav上传文件，但是因为是远程服务器，挂载到本地，实时更新上传会有或多或少的小问题，比如macos下产生的临时文件都会被上传上去，于是我用github仓库做了中转，先上传到github私有仓库，然后在服务器上定时每分钟拉取最新的数据，感觉相对更好一点，如果可以的话是不是可以go集成git，添加一个功能，当然只是一个设想，真心觉得大佬的项目不错。
go-git仓库：https://github.com/go-git/go-git

**Comment by gaowei-space** [comment]
@soft98-top 棒！

我也是用的github中转，博客发布不是那么频繁，目前是在有要发布的文章时用 ansible 拉取一下。推荐的 go-git 我瞅瞅
